ansible/group_vars/dockermanager-hosts.yml

```diff
@@ -1,8 +1,9 @@
 docker_use_ipv4_nic_mtu: True
 
-kubernetes_kube_version: 1.7.0
+kubernetes_kube_version: 1.7.4
 
-idr_jupyter_hub_image: "imagedata/jupyterhub-githubauth:0.4.0"
-idr_jupyter_notebook_image: "imagedata/jupyter-docker:develop"
-idr_jupyter_notebook_repo: https://github.com/IDR/idr-notebooks.git
-idr_jupyter_notebook_repo_version: "0.5.0"
+idr_prepull_docker_images:
+- "imagedata/jupyterhub-githubauth:latest"
+- "imagedata/jupyter-docker:develop"
```

Comment:
Time to tag then? We could call the current state `0.5.0` and then when the remaining 2-3 PRs are merged, `0.6.0`.

Reply:
Either that, or just go ahead and tag the version we'll actually use?

Reply:
`imagedata/jupyter-docker:0.5.0` is getting built as we speak. We can probably update this once the build is completed.